app/services/air_now/import_data.rb

```diff
@@ -4,8 +4,15 @@ def initialize(http_client: Http.new)
   end
 
   def call
+    current_utc = DateTime.now.new_offset(0).beginning_of_hour - 1.hour
```

Comment:
I would use `Time.current` just to be on a safe side.

https://thoughtbot.com/blog/its-about-time-zones